### ISSUE TITLE
[New Rule] Check Yoda Conditions

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#16]
## Explanation

It's way more common to have the variable first.

## Example

```python
# Bad
"Yoda" == i_am

# Good
i_am == "Yoda"
```
